automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -xeuE
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
```

Comment:
nit:
maybe add "Usage:"

Reply:
not sure what you mean, the script is no different from other automation scripts: one simply runs the script

Reply:
just because you wrote lines 11 and 12
so just meant to add "title"
```
Usage:
yum -y install automation/check-patch.packages
....
```

Reply:
ack. got it. thanks!